common/amazon.ts

```diff
@@ -71,33 +73,52 @@ export async function category(url: string) {
   debug.log('Detected category', 'debug')
 
   const categoryObj: Category = {
-    name: $('.bxw-pageheader__title h1').text().trim(),
+    name: $('#nav-search-label-id').text().trim(),
     link: url,
     list: [],
     node
-  }
+  }  
 
   const topRated = $('.octopus-best-seller-card .octopus-pc-card-content li.octopus-pc-item').toArray()
   
-  // @ts-expect-error
-  categoryObj.list = topRated.map(() => {
-    const item = $(this).find('.octopus-pc-item-link')
-    const asin = item.attr('href').split('/dp/')[1].split('?')[0].replace(/\//g, '')
-    const name = item.attr('title')
-    const priceFull = $(this).find('.octopus-pc-asin-price').text().trim()
-    const price = priceFormat(priceFull.replace(/[a-zA-Z]/g, ''))
-
-    return {
+  for(let i = 0; i < topRated.length; i++) {
```

Comment:
Let's maybe change this to `topRated.forEach()`. It's mostly a stylistic choice but I do prefer it

Reply:
Fixed.